src/OperationalInsights/OperationalInsights/Models/DataSourcePropertiesPerKind/PSCustomLogDataSourceProperties.cs

```diff
@@ -207,4 +216,26 @@ public class CustomLogExtraction
 
         #endregion
     }
+
+    public class RegexDelimiterJsonConverter : JsonConverter
```

Comment:
If this serialization is broken in your swagger spec and/or .Net SDK, you must fix it there, and then move this fix into PowerShell.  Fixing it only in PowerShell leaves it broken in other ploaces that consume this swagger spec.

Reply:
okay

Reply:
its not documented in our swagger. the api expects a generic raw json.